### DISCUSSION TITLE
How to set red border stroke for an `<Entry>`?

### DISCUSSION THREAD

**Discussion by MartyIX** [dotnet/maui#7491] [Q&A]
I would like to make something like this https://www.w3schools.com/csS/tryit.asp?filename=trycss_border-color1 (the first red box) but for an `<Entry>` in MAUI XAML. The idea is to change the border color of an entry when the entry does not pass validation rules. 

Is that possible? Does anybody know please?

Thanks!

**Comment by JairoMarques** [comment]
Hi, i'm looking for something similar but without success.

I've tried to use Borde like this code below:
`<Border Grid.Row="2">
    <control:BorderlessEntry
        x:Name="UserCodeEntry"
        PlaceholderColor="{AppThemeBinding Light={DynamicResource Gray600}, Dark={DynamicResource Gray300}}"
        Placeholder="User Code"
        Style="{StaticResource EntryStyle}"
        Text="{Binding UserCode}" />
</Border>`

but in this case will have 2 borders. One for entry and other for Border itself
